### ISSUE TITLE
Races on create

### ISSUE THREAD

**Issue by smoser** [anuvu/disko#103]
A user of disko is creating devices in parallel and in some cases we hit a race where the creation of an LV fails due to already existing.

2 things disko could do:
 * nothing - creation should fail if you try to create a LV that already exists. (just as 'mkdir' fails if the entry exists).  Here we should probably return a specific error to allow the caller to handle.
 * create, then if creation failed and the device did exist matching characteristics, return success.

**Comment by smoser** [comment]
I think it makes best sense to do the first... just return an error indicating the volume exists like `os.ErrExist`.

**Comment by hallyn** [comment]
Yes, a caller might expect that if it returns success, it must be the first creator and first mounter, so then might do some one-time setup.

I suppose you could add an optional "ignore-eexist" flag, but i think it's better for you to just pass back eexist and let the caller handle it.
